aiohttp/web_response.py

```diff
@@ -268,53 +304,63 @@ def last_modified(self, value):
         elif isinstance(value, str):
             self.headers[hdrs.LAST_MODIFIED] = value
 
-    def _generate_content_type_header(self, CONTENT_TYPE=hdrs.CONTENT_TYPE):
-        params = '; '.join("%s=%s" % i for i in self._content_dict.items())
+    def _generate_content_type_header(
+        self,
+        CONTENT_TYPE: str=hdrs.CONTENT_TYPE,
+    ) -> None:
+        params = '; '.join(
+            ("%s=%s" % (k, v) for k, v in self._content_dict.items()),
+        )
         if params:
-            ctype = self._content_type + '; ' + params
+            ctype = self.content_type + '; ' + params
```

Comment:
Not really sure why variable is `Optional[str]` and attr getter from super returns `str`. Can't tell yet which one is correct.